sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -247,22 +255,20 @@ export class SecretClient {
     secretName: string,
     value: string,
     options?: SetSecretOptions
-  ): Promise<Secret> {
+  ): Promise<KeyVaultSecret> {
     if (options) {
       const unflattenedProperties = {
         enabled: options.enabled,
         notBefore: options.notBefore,
-        expires: options.expires
+        expires: options.expiresOn
```

Comment:
`expiresOn`

Reply:
no no, this is what gets sent to the service